examples/semi_supervised/plot_label_propagation_digits_active_learning.py

```diff
@@ -70,7 +70,9 @@
         lp_model.label_distributions_.T)
 
     # select five digit examples that the classifier is most uncertain about
-    uncertainty_index = uncertainty_index = np.argsort(pred_entropies)[-5:]
+    uncertainty_index = np.argsort(pred_entropies)[::-1]
+    uncertainty_index = uncertainty_index[
+        np.in1d(uncertainty_index, unlabeled_indices)][:5]
```

Comment:
Sorry, this comment didn't submit:
is it possible this expression will return fewer than 5 results? Does this need to be handled below?